src/main/java/net/geforcemods/securitycraft/items/CodebreakerItem.java

```diff
@@ -14,6 +26,28 @@ public CodebreakerItem() {
 		super(new Item.Properties().maxStackSize(1).defaultMaxDamage(3).group(SecurityCraft.groupSCTechnical));
 	}
 
+	@Override
+	public ActionResult<ItemStack> onItemRightClick(World world, PlayerEntity player, Hand hand) {
+		ItemStack codebreaker = player.getHeldItem(hand);
+
+		if (!world.isRemote) {
+			if (hand == Hand.MAIN_HAND && player.getHeldItemOffhand().getItem() == SCContent.BRIEFCASE.get()) {
+				ItemStack briefcase = player.getHeldItemOffhand();
+
+				if(!ConfigHandler.CONFIG.allowCodebreakerItem.get()) {
+					PlayerUtils.sendMessageToPlayer(player, ClientUtils.localize(SCContent.BRIEFCASE.get().getTranslationKey()), ClientUtils.localize("messages.securitycraft:codebreakerDisabled"), TextFormatting.RED);
+					return ActionResult.resultFail(codebreaker);
+				}
+				else if (new Random().nextInt(3) == 1) {
+					SecurityCraft.channel.sendToServer(new OpenGui(SCContent.cTypeBriefcaseInventory.getRegistryName(), briefcase.getDisplayName()));
```

Comment:
You're running this code on the server (line 33, `if(!world.isRemote)`), but are trying to send a packet to the server. That's not going to work. Check line 65-77 of the `OpenGui` packet to see what you should do here.

Reply:
Thing is, it did work and it opened the gui when the codebreaker succeeded. Will do what you said though

Reply:
It works in a singleplayer world because the client is physically present, but it does not work on a dedicated server.